synapsis/src/main/java/com/soen/synapsis/appuser/connection/ConnectionService.java

```diff
@@ -0,0 +1,38 @@
+package com.soen.synapsis.appuser.connection;
+
+import com.soen.synapsis.appuser.AppUser;
+import com.soen.synapsis.appuser.Role;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ConnectionService {
+    private final ConnectionRepository connectionRepository;
+
+    public ConnectionService(ConnectionRepository connectionRepository) {
+        this.connectionRepository = connectionRepository;
+    }
+
+    public String makeConnection(AppUser requester, AppUser receiver) {
+        if (requester.getRole() != Role.CANDIDATE) {
```

Comment:
Our intended business logic is that anybody except admins can connect to each other.

Reply:
Modified code logic so that admins cannot make and receive connections.